fili-core/src/main/java/com/yahoo/bard/webservice/table/PhysicalTable.java

```diff
@@ -58,23 +75,19 @@
      * should likely use their own serialization strategy so as to not hit this defaulting behavior.
      *
      * @param logicalName  Logical name to lookup in physical table
+     *
      * @return Translated logicalName if applicable
      */
     String getPhysicalColumnName(String logicalName);
 
     /**
-     * Determine whether or not this PhysicalTable has a mapping for a specific logical name.
+     * Getter for active column intervals.
      *
-     * @param logicalName  Logical name to check
+     * @param constraints data constraint containing columns and api filters
      *
-     * @return True if contains a non-default mapping for the logical name, false otherwise
-     *
-     * @deprecated This may no longer be needed
+     * @return tableEntries map of column to set of available intervals
```

Comment:
This JavaDoc isn't accurate. The method is returning a `SimplifiedIntervalList`, not a map.